proposed/http-client/http-client-meta.md

```diff
@@ -54,8 +53,8 @@ principle). The PSR does not try to restrict nor define how to configure HTTP cl
 An alternative approach would be to pass configuration to the client. That approach
 would have a few drawbacks:
 
-* Configuration must be defined by the PSR;
-* All clients must support the defined configuration;
+* Configuration must be defined by the PSR.
+* All clients must support the defined configuration.
```

Comment:
Same comment here as for another bullet list: these are all self-contained sentences.